lib/components/LazyImage.vue

```diff
@@ -82,7 +89,7 @@ export default {
 
   computed: {
     hasSlot () {
-      return this.$slots.default
+      return this.$slots.caption
```

Comment:
Wollen wir caption wirklich in LazyImage verwalten?

Reply:
Hier stimmte die Überprüfung nicht.`figcaption` soll ja nur gerendert werden wenn auch im `caption` Slot ein Element gesetzt ist.
```html
      <figcaption v-if="hasSlot || caption">
```

Reply:
stimmt, der slot ist ja named